Remove Replit in "Calling APIs" > "Managing Asynchronous Calls" (#163)
Ada-Developers-Academy/core-unit-3: calling-apis/managing-asynchronous-calls.md

### !end-challenge
<!-- prettier-ignore-end -->

Raffy chose to create a new function `getLocationFromQuery` to manage calling the existing functions `findLatitudeAndLongitude` and `findLocation`. He could have called the second method directly from inside the first with `.then` chaining, but doing so would have linked these two functions together, making it harder to use one without the other.

To use `findLatitudeAndLongitude` from another function and pass on its `response` data to another network call, we needed to make a couple updates in `findLatitudeAndLongitude`: 

> - `return` the result of the `axios` call which returns the promise chain the axios call creates.

Reviewer: This sentence is a little challenging for me to follow, but maybe because it's Friday afternoon and the sun is shining. 

Trying to think how it could be reworded.

"`return` the result of the `axios` call because we'll need to access the promise chain the axios call creates."

Not sure if that's any more clear. So maybe we can refer to the linked repo for line numbers since code blocks in Learn don't have line numbers instead?

```suggestion
- `return` the result of the `axios` call which returns the promise chain the axios call creates (see line 86 in the linked repo)
```

> - `return` the result of the `axios` call which returns the promise chain the axios call creates.

Author: I think this can be clearer, but I'm a little concerned about putting an exact line number in the lesson that could get out of sync. How do you feel about:

```suggestion
- `return` the result of the `axios` call. This returns the promise chain created by the network call and chained `.then` blocks. Look for the `return` statement in `findLatitudeAndLongitude` that starts with `return axios.get(...`.
```

> - Inside the `.then` block, we unpack the `latitude` and `longitude` from the `response` object and return just those values we want in a new Object. This allows us to pass these values to a new `.then` block wherever we invoke the function.

Reviewer: ```suggestion
- Inside the `.then` block, we unpack the `latitude` and `longitude` from the `response` object and return just those values we want in a new Object. This allows us to pass these values to a new `.then` block wherever we invoke `findLatitudeAndLongitude`.
```

How do you feel about replacing "the function" with the name itself to be extra clear. I feel like this lesson could be challenging to wrap your mind around if it's one's first time working with async code.
